### PR TITLE
React radio buttons for private events

### PR DESCRIPTION
**Pull request by vpatel90**
Added react components to new event form, migrated open_invite col to events table, modified events controller to accept new params.

### CODE REVIEW

File: app/controllers/events_controller.rb [hunk 1]
```diff
@@ -18,6 +18,7 @@ def edit
   end
 
   def create
+    binding.pry
```
Comment (jah2488) [inline app/controllers/events_controller.rb]: 🕵 This needs to get removed.

Comment (vpatel90) [inline app/controllers/events_controller.rb]: saw that right as I pushed smh

**Comment by reckocp** [comment]
👍

**Comment by VanAnderson** [comment]
:+1:

**Pull request merged**